Content.Server/Administration/Commands/SetMindCommand.cs

```diff
@@ -0,0 +1,82 @@
+#nullable enable
+using Content.Server.GameObjects.Components.Mobs;
+using Content.Server.Mobs;
+using Content.Server.Players;
+using Content.Shared.Administration;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Administration.Commands
+{
+    [AdminCommand(AdminFlags.Admin)]
+    class SetMindCommand : IClientCommand
+    {
+        public string Command => "setmind";
+
+        public string Description => Loc.GetString("Transfers a mind to the specified entity. The entity must have a MindComponent.");
+
+        public string Help => Loc.GetString("Usage: {0} <entityUid> <username>", Command);
+
+        public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
+        {
+            if (args.Length != 2)
+            {
+                shell.SendText(player, Loc.GetString("Wrong number of arguments."));
+                return;
+            }
+
+            if (!int.TryParse(args[0], out var entityUid))
+            {
+                shell.SendText(player, Loc.GetString("EntityUid must be a number."));
+                return;
+            }
+
+            var entityManager = IoCManager.Resolve<IEntityManager>();
+
+            var eUid = new EntityUid(entityUid);
+
+            if (!eUid.IsValid() || !entityManager.EntityExists(eUid))
+            {
+                shell.SendText(player, Loc.GetString("Invalid entity ID."));
+                return;
+            }
+
+            var target = entityManager.GetEntity(eUid);
+
+            if (!target.TryGetComponent<MindComponent>(out var mindComponent))
+            {
+                shell.SendText(player, Loc.GetString("Target entity does not have a mind (did you forget to make sentient?)"));
+                return;
+            }
```

Comment:
making this `ensurecomponent` would make more sense IMO

Reply:
Stylistically sure, but it means looking up the MindComponent in an entity twice.

Reply:
The reason I didn't do EnsureComponent is because I'm assuming that Make Sentient does important stuff so the object can move

Reply:
Confirmed, Make Sentient *does* do important stuff so the object can move
So I'm not going to go apply a MindComponent